packages/regl-worldview/src/stories/SDFGenerator.stories.js

```diff
@@ -0,0 +1,42 @@
+//  Copyright (c) 2018-present, GM Cruise LLC
+//
+//  This source code is licensed under the Apache License, Version 2.0,
+//  found in the LICENSE file in the root directory of this source tree.
+//  You may not use this file except in compliance with the License.
+
+import { storiesOf } from "@storybook/react";
+import React from "react";
+
+import SDFGenerator from "../utils/SDFGenerator";
+
+const DEFAULT_ATLAS_CONFIG = {
+  fontSize: 14,
+  fontFamily: "sans-serif",
+  charSet: new Set(
+    " !\"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~".split("")
```

Comment:
Not sure if it's worth it, but I would add a couple of icons here too.

Reply:
Good call, I added a few